diff --git a/node_modules/react-native-get-sms-android/android/build.gradle b/node_modules/react-native-get-sms-android/android/build.gradle
index 0de40fc..7f606ac 100644
--- a/node_modules/react-native-get-sms-android/android/build.gradle
+++ b/node_modules/react-native-get-sms-android/android/build.gradle
@@ -32,11 +32,7 @@ android {
 
 repositories {
     google()
-    maven {
-        // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
-        url "$rootDir/../node_modules/react-native/android"
-    }
-    jcenter()
+    mavenCentral()
 }
 
 dependencies {
